docs/source/09_development/03_commands_reference.md

```diff
@@ -119,7 +119,7 @@ v0.17.6
 
 kedro allows teams to create analytics
 projects. It is developed as part of
-the Kedro initiative at QuantumBlack.
+the Kedro initiative.
```

Comment:
Might I suggest this text becomes: 
"Kedro is a Python framework for\n"
"creating reproducible, maintainable\n"
"and modular data science code."